js/src/dapps/registry/events/events.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component, PropTypes } from 'react';
+import { Card, CardHeader, CardActions, CardText } from 'material-ui/Card';
+import Toggle from 'material-ui/Toggle';
+
+import { IdentityIcon } from '../parity.js';
+import styles from './events.css';
+import bytesToHex from '../../../api/util/bytes-array-to-hex';
+
+const inlineButton = {
+  display: 'inline-block',
+  width: 'auto',
+  marginRight: '1em'
+};
+
+const renderOwner = (owner) => (
+  <span className={ styles.owner }>
+    <IdentityIcon inline center address={ owner } />
+    <code>{ owner }</code>
+  </span>
+);
+
+const renderReserved = (e) => (
+  <p key={ e.key } className={ styles.reserved }>
+    { renderOwner(e.parameters.owner) }
+    { ' ' }
+    <abbr title={ e.transaction }>reserved</abbr>
+    { ' ' }
+    <code>{ bytesToHex(e.parameters.name) }</code>
+  </p>
+);
+
+const renderDropped = (e) => (
+  <p key={ e.key } className={ styles.dropped }>
+    { renderOwner(e.parameters.owner) }
```

Comment:
I think I would prefer JSX instead of function calls here:
`<Owner owner={e.parameters.owner} />`
` const RenderOwner = ({owner}) => ( ...unchanged... );`

For me it would be easier to scan throught structure more quickly: It would be just another component displayed along with `abbr` and `code` tags.

A matter of taste, though.